packages/components/src/ResourceList/ResourceList.stories.js

```diff
@@ -259,7 +259,7 @@ export function FilteredResourceList(props) {
 			toolbar={{
 				name: {
 					label: t('FILTER', {
-						defaultValue: 'Find a {{type, lowercase}}',
+						defaultValue: 'Filter:',
 						type: props?.renderAs?.name,
```

Comment:
Why a `type` parameter if the defaultValue is 'Filter' ?

Reply:
removed type param